src/components/AvatarWithImagePicker.js

```diff
@@ -124,6 +125,17 @@ class AvatarWithImagePicker extends React.Component {
         this.setState({isErrorModalVisible: false});
     }
 
+    /**
+     * Check if the attachment extension is allowed.
+     *
+     * @param {Object} image
+     * @returns {Boolean}
+     */
+    isValidExtension(image) {
+        const fileExtension = lodashGet(FileUtils.splitExtensionFromFileName(image.name), 'fileExtension', '');
```

Comment:
What I mean here is if `image` is null for any reason you'll still get an error.

```suggestion
      const {fileExtension} = FileUtils.splitExtensionFromFileName(lodashGet(image, 'name', ''));
```

Reply:
Ahh, my bad.